### PR TITLE
Global Websites nav

### PR DESCRIPTION
**Pull request by denlight**

### CODE REVIEW

File: src/assets/cru-scss/_global-website-list.scss [hunk 1]
```diff
@@ -1,4 +1,23 @@
+/***** TESTING *****/
```
Comment (OzzieOrca) [inline src/assets/cru-scss/_global-website-list.scss]: Does this still need to be here? There's also a lot of whitespace below this class...

File: src/assets/cru-scss/_global-website-list.scss [hunk 2]
```diff
@@ -44,53 +68,41 @@
...
+    padding-right: 2.12%;
     background: rgba(0, 0, 0, 0.25);
 
-    @include mq(550px, max) {
-        margin-bottom: 0;
-        padding-top: 0;
-        padding-bottom: 0;
-        margin-left: 0;
-    }
+    /* @include mq(1000px, max) {
```
Comment (OzzieOrca) [inline src/assets/cru-scss/_global-website-list.scss]: Does this need to be here still?

File: src/assets/cru-scss/_global-website-list.scss (hunk 1 above)
Comment (adammeyer) [inline src/assets/cru-scss/_global-website-list.scss]: I agree, this should be cleaned up.  @j2trumpet Can you give input on this.  What lines can be removed?

**Comment by adammeyer** [comment]
@OzzieOrca You can close 225, this is going to replace that.

**Comment by j2trumpet** [comment]
@denlight Lines 1-12 and 77-80 of src/assets/cru-scss/_global-website-list.scss can be removed. Sorry that got committed.

File: src/assets/cru-scss/_global-website-list.scss [hunk 3]
```diff
@@ -1,15 +1,3 @@
-/***** TESTING *****/
-.globalWebsites--is-open {
-  background-image: url(https://www.cru.org/images/globalWebsites--huge.jpg) !important;
```
Comment (adammeyer) [inline src/assets/cru-scss/_global-website-list.scss]: @j2trumpet Does this background still need to be here?  It's not missing the image.

**Pull request merged**